### DISCUSSION TITLE
Creating new test file

### DISCUSSION THREAD

**Discussion by rishujam** [oppia/oppia-android#4770] [Q&A (Other)] [answered]
1. Can't differentiate between androidTest and test (In my case I want to add a test file which do not require any android component)
![Screenshot 2022-11-28 000250](https://user-images.githubusercontent.com/74773876/204153237-177220a3-8d47-498d-98b6-a4e8fbb27247.png)
2. While doing edit configuration can't see Android JUnit (only JUnit) are they the same?
![Screenshot 2022-11-28 000355](https://user-images.githubusercontent.com/74773876/204153280-6dcf70aa-22ac-4fab-b2d2-e006a9573d90.png)

**Comment by BenHenning** [comment, marked as answer]
Which version of Android Studio are you using @rishujam? There's an issue with versions later than Bumblebee where AGP won't properly handle multi-module projects like ours (specifically, the way we set up shared tests)--see #4572.

Currently, all app module tests are either under sharedTest (which means they can run with both Robolectric and Espresso), or test (which means they are Robolectric only). What's your use case for not requiring any Android components? That's generally rare in the UI module since even hooking into the Dagger graph requires Android dependencies.
